<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="cwidget">
   <widget class="PlotWidget" name="gview">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>681</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pBut">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txtBrw">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>620</y>
      <width>701</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="randCheck">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>60</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RandCheck</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sizeSpin">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>110</y>
      <width>48</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="pixelModeCheck">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>80</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PixelModeCheck</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="gview_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>681</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="gview_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>681</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
